### DISCUSSION TITLE
Target gene of Transcription factor(TF)

### DISCUSSION THREAD

**Discussion by summer-tao** [stuart-lab/signac#572] [Ideas]
Hi, Thank you for providing such a useful tool!

I have a question. From my result, I identified TF Rorc enrichment on my cluster 1, which matches my biology knowledge. May I conduct a type of analysis to identify the target gene of Rorc, please? For example, there is most Rorc binding motif enrichment on the detected Cd3d fragment, so maybe Cd3d is the target gene of Rorc. But there are also some other targets.

I know, if we want to do this analysis, we need to think about the activity differences between different genes. Just an idea, I think if we find the target genes which has both higher activity and higher enrichment of this TF motif may be the possible target gene.

Best,
Tao.

**Comment by lzh-fugwaa** [comment]
Hello, I also have the same need. Could you please tell me if you have any good solutions?
